google/cloud/security/common/data_access/forseti_system_dao.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides the data access object (DAO) for Forseti system management."""
+
+from google.cloud.security.common.data_access import dao
+# pylint: disable=line-too-long
+from google.cloud.security.common.data_access.sql_queries import cleanup_tables_sql
+from google.cloud.security.common.util import log_util
+
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class ForsetiSystemDao(dao.Dao):
+    """Data access object (DAO) for Forseti system management.
+        Args:
```

Comment:
Since we are making other changes, might as well ask you to do this, too.  Can you please add a new line to separate the Args section from the method doc summary line?

Also, can you shift the indentation of the Args section by 4 spaces?  Like this:

```
class ForsetiSystemDao(dao.Dao):
    """Data access object (DAO) for Forseti system management.

    Args:
        global_configs (dict): Global config - used to lookup db_name
    """
```